amgut/lib/data_access/ag_data_access.py

```diff
@@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
         return return_res
 
     def getAnimalParticipants(self, ag_login_id):
-        results = self._sql.execute_proc_return_cursor(
-            'ag_get_animal_participants', [ag_login_id])
-
-        return_res = [row[0] for row in results]
-        results.close()
-        return return_res
+        sql = """SELECT participant_name FROM ag_animal_survey
```

Comment:
Yeah, I think that makes sense to do. @wasade, thoughts? The main reason I'm concerned is that the function `getAnimalParticipants` might be used in places where it is assumed that a set of operations can be applied equally to each item in the output, which might not be the case if there is more than one source for the data.